src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1052,7 +1047,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         if (_terminal->IsSelectionActive())
         {
             _terminal->SetBlockSelection(!_terminal->IsBlockSelection());

```

Comment:
idle thought: maybe we file a follow up to change the shape of the markers when we're in block selection mode? idk.

Reply:
Oh yeah. Dustin and I were chatting about that the other day. it would be cool if we used some kind of 'L' shape for that.

Definitely a follow-up and something to mention/show in the spec.